xbmc/utils/posix/Shm.h

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
I think >95% all people seeing this file will struggle expanding "shm" to shared memory.